ibm_catalog.json

```diff
@@ -96,6 +109,66 @@
               },
               "key": "region",
               "required": true
+            },
+            {
+              "key": "existing_secrets_manager_instance_crn"
+            },
+            {
+              "key": "existing_secrets_manager_cert_crn"
+            },
+            {
+              "key": "cert_common_name"
+            },
+            {
+              "key": "root_ca_name"
+            },
+            {
+              "key": "root_ca_common_name"
+            },
+            {
+              "key": "intermediate_ca_name"
+            },
+            {
+              "key": "certificate_template_name"
+            },
+            {
+              "key": "existing_subnet_names"
```

Comment:
There might be a custom catalog type for subnets. Can you check? (as part of https://github.ibm.com/GoldenEye/issues/issues/11555)